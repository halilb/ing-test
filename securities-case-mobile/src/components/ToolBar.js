import React from 'react';
import { Text, View, StyleSheet } from 'react-native';
import PropTypes from 'prop-types';

const ToolBar = ({ title }) => (
  <View style={styles.container}>
    <Text style={styles.title}>{title}</Text>
  </View>
);

const styles = StyleSheet.create({
  container: {
    height: 64,
    alignItems: 'center',
    justifyContent: 'center',
    backgroundColor: '#44D4D7',
  },
  title: {
    fontSize: 16,
    fontWeight: 'bold',
    color: 'white',
  },
});

ToolBar.propTypes = {
  title: PropTypes.string.isRequired,
};

export default ToolBar;
