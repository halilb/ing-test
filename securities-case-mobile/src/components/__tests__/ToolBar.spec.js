import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import ToolBar from '../ToolBar';

it('renders a ToolBar correctly', () => {
  const snapshot = renderer.create(<ToolBar title={'Instruments'} />).toJSON();
  expect(snapshot).toMatchSnapshot();
});
