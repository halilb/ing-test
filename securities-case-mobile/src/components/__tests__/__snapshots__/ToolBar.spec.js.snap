// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a ToolBar correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#44D4D7",
      "height": 64,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "white",
        "fontSize": 16,
        "fontWeight": "bold",
      }
    }
  >
    Instruments
  </Text>
</View>
`;
